src/codegen.cpp

```diff
@@ -5482,8 +5483,8 @@ static void init_julia_llvm_env(Module *m)
         "jl_value_t",
         julia_h,
         71, // At the time of this writing. Not sure if it's worth it to keep this in sync
-        sizeof(jl_value_t)*8,
-        __alignof__(jl_value_t)*8,
+        0 * 8, // sizeof(jl_value_t) * 8,
```

Comment:
@Keno: I think we've had a conversation about before, but I don't remember the result. Is DWARF going to dislike a declared size of 0 bits?